src/typeahead/typeahead.ts

```diff
@@ -241,6 +250,10 @@ export class NgbTypeahead implements ControlValueAccessor,
       this._windowRef.instance.id = this.popupId;
       this._windowRef.instance.selectEvent.subscribe((result: any) => this._selectResultClosePopup(result));
       this._windowRef.instance.activeChangeEvent.subscribe((activeId: string) => this.activeDescendant = activeId);
+
+      if (this.container === 'body') {
+        window.document.querySelector(this.container).appendChild(this._windowRef.location.nativeElement);
```

Comment:
Current `Renderer2` interface doesn't have any method(s) to query nodes and / or call an arbitrary method on a DOM node: https://github.com/angular/angular/blob/10897d64738484036443f6614ad3cbef095da0d5/packages/core/src/render/api.ts#L147-L191